body.html

```diff
@@ -4,10 +4,20 @@
     <p>Hello,</p>
 
     <p>
-      Please find attached an updated JSON file listing the BOD 18-01 web hosts
-      that require authentication via client certs.
+      Please find attached an updated CSV file listing the BOD 18-01 web hosts
+      that require authentication via client certificates.
     </p>
 
-    <p>Thank you</p>
+    <p>
+      Thank you,<br />
+      The NCATS team
+    </p>
+
+    <p>
+      National Cybersecurity Assessments and Technical Services (NCATS)<br />
+      National Cybersecurity and Communications Integration Center<br />
+      U.S. Department of Homeland Security<br />
```

Comment:
Shouldn't we change our signature block to something without NCCIC and DHS, that mentions CISA?  NCATS can stay for now I think.

Reply:
Good point, @dav3r.  How do you like the signature block after commit 993d291.

Reply:
That looks great to me!  Good call on updating cyhy-mailer similarly.